packages/quantic/force-app/main/default/lwc/quanticResultList/e2e/pageObject.ts

```diff
@@ -0,0 +1,90 @@
+import type {Locator, Page, Request} from '@playwright/test';
+import {isUaClickEvent} from '../../../../../../playwright/utils/requests';
+
+export class ResultListObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
+
+  get performSearchButton(): Locator {
+    return this.page.locator('c-action-perform-search button');
+  }
+
+  get resultList(): Locator {
+    return this.page.locator('c-quantic-result-list');
+  }
+
+  get resultLinks(): Locator {
+    return this.resultList.locator('c-quantic-result-link a');
+  }
+
+  getResultLink(index: number): Locator {
+    return this.resultLinks.nth(index);
+  }
+
+  async preventNavigationFromResultLink(index: number): Promise<void> {
+    return this.getResultLink(index).evaluate((el) => {
+      el?.addEventListener?.('click', (event) => {
+        event.preventDefault();
+      });
+    });
+  }
+
+  async performSearch(): Promise<void> {
+    await this.performSearchButton.click();
+  }
```

Comment:
Same, can be removed, better to use this instead: https://github.com/coveo/ui-kit/blob/dc880bbfb8b1d3160be733236aa5543708fdcbd9/packages/quantic/playwright/page-object/searchObject.ts#L24